ignite/engine/engine.py

```diff
@@ -138,7 +171,7 @@ def on(self, event_name, *args, **kwargs):
         """Decorator shortcut for add_event_handler
 
         Args:
-            event_name (Events): event to attach the handler to
+            event_name: event to attach the handler to
```

Comment:
nit: same as said Aly on `add_event_handler`: 
event from `ignite.engine.Events` or any event_name added by `register_events` to attach the handler to